crypto/src/main/java/org/springframework/security/crypto/bcrypt/BCryptPasswordEncoder.java

```diff
@@ -99,6 +99,10 @@ public BCryptPasswordEncoder(BCryptVersion version, int strength, SecureRandom r
 	}
 
 	public String encode(CharSequence rawPassword) {
+		if (rawPassword == null) {
+			throw new IllegalArgumentException("Raw password cannot be null");
```

Comment:
To follow Spring Security conventions, please change `Raw password` to match the variable name of `rawPassword`.

Reply:
Please add a test for this use case

Reply:
done